fullstack/README.md

```diff
@@ -0,0 +1,64 @@
+This is a coding exercise to help assess candidates looking to join our team at Condé Nast International as FullStack Developers.
+
+The test is intended to be completed at home in no more than 1-2 hours. It is not timed, so if you want to take more time to showcase your coding skills and ability you may do so (note this is not mandatory, nor expected).
+
+If you haven't finished in that time, you can send us your progress - we don't want you to spend days on this. We understand that your time is valuable, and appreciate any time that you contribute towards building a strong team here.
+
+If you **really cannot spare the time**, you may want to take a look at Option 2.
+
+## Option 1
+
+Build a JavaScript application that shows the latest news from the United Kingdom, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see the latest **UK news** and give the ability to the user to filter them by categories. Given the fact that this is a FullStack challenge, we are expecting a Frontend application and a Backend application that communicate with each other.
```

Comment:
When we say "filter by categories", do we mean search by keywords? I don't think the News API has a concept of "categories" per se.

Reply:
I could be more clear pointing them to: https://newsapi.org/docs/endpoints/sources

Reply:
I think that endpoint is for filtering by publisher (which we could ask them to do?).

For filtering by keywords, this endpoint seems more useful: https://newsapi.org/docs/endpoints/everything